### ISSUE TITLE
sonarr and radarr issues

### ISSUE THREAD

**Issue by M0UL** [SynoCommunity/spksrc#3474]
### Setup
_Package Name:_ Sonarr
_Package Version:_ 20180303-13
_Package Name:_ Radarr
_Package Version:_  20180303-6

_NAS Model:_ds415+
_NAS Architecture:_ avaton 
_DSM version:_6.1.7

### Expected behavior
sonarr and radarr should restart when pressing the restart button in there web interfaces.
### Actual behavior
the apllication just stop without restarting and  restarting it from the packages manger does not work either. i have to restart the whole nas for it work. this applies to both raddarr and sonarr.
### Steps to reproduce
_1._ open the sonarr of radarr web ui
_2._ press the  restart button 
_3._ that it just stops sonarr/radarr

**Comment by BenjV** [comment]
Stopping and starting Sonarr via the package Centre works just fine.
So why bother to do it within Sonarr?

And if you insist on that functionality from within Sonarr, you should ask the question in de Sonarr website, not here,

**Comment by M0UL** [comment]
it only does not work here. every where else it works fine.

**Comment by BenjV** [comment]
I have tested it an there no problem over here.
Sonarr would start fine after I stopped it from within Sonarr.
It takes some time before Sonarr is completely stopped if you do it from within Sonarr so maybe you must be patient.
Especially if Sonarr is busy it may take a while.

Does stopping and starting from the package center work for you?

**Comment by M0UL** [comment]
stoping and than restarting it from the package center works fine.

**Comment by BenjV** [comment]
If you still have this problem you should ask it on the Sonarr website, because if it won't stop from within Sonarr then it is clearly a Sonarr problem and not a package problem.

**Comment by houndtt** [comment]
hey @M0UL, is this issue still happening for you? the only similar situation I had was with Sonarr a while back and for me I was able to resolve it by resetting some of the advanced settings under media management (see my post here: https://github.com/SynoCommunity/spksrc/issues/3859#issuecomment-578547469)

as for Radarr, I did for a brief time experience this (my report here: https://github.com/Radarr/Radarr/issues/4931) but after resetting some of the same media management configurations to the defaults I was fine. with the recent upgrade of radarr to v3 I haven't seen any issues. what has been your experience with the new version?
